app/src/main/java/org/mozilla/fenix/addons/InstalledAddonDetailsFragment.kt

```diff
@@ -44,17 +46,28 @@ class InstalledAddonDetailsFragment : Fragment() {
             addon = AddonDetailsFragmentArgs.fromBundle(requireNotNull(arguments)).addon
         }
 
-        return inflater.inflate(R.layout.fragment_installed_add_on_details, container, false).also {
-            bindUI(it)
-        }
+        _binding = FragmentInstalledAddOnDetailsBinding.inflate(
+            inflater,
+            container,
+            false
+        )
+
+        bindUI(binding)
+
+        return binding.root
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        bindAddon(view)
+        bindAddon(binding)
```

Comment:
It seems unnecessary for us to pass `binding` into `bindUI` and `bindAddon` since `binding` is a property of the class and can just be accessed directly.

Reply:
You are right, sorry bout that. 👍